### ISSUE TITLE
Remove all test failed plugins from the README

### ISSUE THREAD

**Issue by imbsky** [asdf-vm/asdf-plugins#183]
We are currently considering removing all test failed plugins from the README. Do you have the bandwidth to fix so that the test runs correctly again. Whether you have bandwidth or not, we recently started a collaborative, community-driven project called asdf-community for long-term maintenance of asdf plugins, so consider moving your plugins here.
https://github.com/asdf-community/.github

I'm tagging all the authors of the corresponding plugin (in no particular order), but I'd like to hear everyone's opinions.

@Ibotta (`Ibotta/asdf-brig`)
@marciogm (`marciogm/asdf-crystal`)
@mikestephens (`mikestephens/asdf-elasticsearch`)
@vic (`vic/asdf-graalvm`, `vic/asdf-haskell`)
@rkyleg (`rkyleg/asdf-julia`)
@sylph01 (`sylph01/asdf-mongodb`)
@richin13 (`richin13/asdf-neovim`)
@particledecay (`particledecay/asdf-osctl`)
@odarriba (`odarriba/asdf-php`)
@code-lever (`code-lever/asdf-rust`)
@mtatheonly (`mtatheonly/asdf-scala`)
@RykHawthorn (`RykHawthorn/asdf-tflint`)
@ndac-todoroki (`ndac-todoroki/asdf-v`)

**Comment by imbsky** [comment]
CC: @blimmer (`Ibotta/asdf-brig`)

**Comment by imbsky** [comment]
CC: @nickveys, @gabecook (`code-lever/asdf-rust`)

**Comment by imbsky** [comment]
Of course, I can help you run the test correctly again.

**Comment by sylph01** [comment]
I do recognize that there are issues for newer versions of mongodb, but its fix is not trivial, and I cannot work on fixing it right away. As such, please remove `asdf-mongodb` for now, and I will contact you again after fixing it.

**Comment by imbsky** [comment]
@sylph01 OK. I'm looking forward to your contact.

**Comment by odarriba** [comment]
I currently don't have bandwidth to maintain `asdf-php` in any of the ways I would like to (new versions, optimisations, PHP extensions or fixing the tests), so I would agree to transfer it to this community if it starts moving and maintaining this kind of projects.

**Comment by imbsky** [comment]
@odarriba We already manage a lot of plugins, for example the Standard ML plugin has been transferred by the author. If you okay, I will send you an invitation to prepare for the transfer.

**Comment by odarriba** [comment]
@imbsky let's transfer it then!

**Comment by imbsky** [comment]
@odarriba Sent!

**Comment by marciogm** [comment]
hey @imbsky.
I'm willing to transfer the `asdf-crystal` as well, could you send a invitation?

**Comment by nickveys** [comment]
@imbsky Just got asdf-rust fixed up, all green!  Thanks for the reminder.

**Comment by imbsky** [comment]
@marciogm Sent!

**Comment by imbsky** [comment]
@nickveys Thank you! Really quick work!

**Comment by imbsky** [comment]
I fixed the asdf-crystal's CI and dealt with all issues.

**Comment by blimmer** [comment]
@imbsky it looks like there's some kind of issue with Travis CI's badge on that repo. The tests are actually passing, even though the badge says that they're not:

<img width="341" alt="Screen Shot 2020-01-26 at 8 27 03 AM" src="https://user-images.githubusercontent.com/630449/73137508-05dc4000-4016-11ea-9b60-d62c0a8f4538.png">

I opened https://github.com/Ibotta/asdf-brig/issues/8 to see if they can fix up the badge, as I no longer have maintainer abilities on that repo.

**Comment by imbsky** [comment]
Thank you for checking! Hmm, that's weird... We've created a great tool for GitHub Actions, so it might be a good idea to move to it. https://github.com/asdf-vm/actions

**Comment by blimmer** [comment]
Sounds good. I opened https://github.com/Ibotta/asdf-brig/issues/9 for them to consider, since I don't have maintainer privs anymore. Thanks!

**Comment by imbsky** [comment]
@blimmer Thank you again for your quick response!

**Comment by ndac-todoroki** [comment]
@imbsky Sorry for being late, I've fixed `asdf-v`. Thanks for the reminder!

**Comment by imbsky** [comment]
@ndac-todoroki Great!

**Comment by mikestephens** [comment]
As a whole I think asdf community is a great thing, I'd be happy to pass `asdf-elasticsearch` off to that

**Comment by imbsky** [comment]
@mikestephens I sent you an invitation!

**Comment by mtatheonly** [comment]
@imbsky Fixed the asdf-scala plugin =)

**Comment by imbsky** [comment]
@mtatheonly Thank you!

**Comment by vic** [comment]
Nice, I'll also try fixing mines this weekend and transfer them.

**Comment by imbsky** [comment]
Awesome! By the way, most plugins have been fixed by the author or community maintainer. Thanks to everyone who helped us.

**Comment by particledecay** [comment]
![Selection_762](https://user-images.githubusercontent.com/685255/73573831-4dadfd80-4442-11ea-9deb-2ade30af65ba.png)
This plugin was renamed to "Talos" per #151. Sorry... should have removed osctl. I can submit another PR to remove it.

**Comment by imbsky** [comment]
@particledecay All right!

**Comment by imbsky** [comment]
I just setup GitHub Actions on asdf-php in https://github.com/asdf-community/asdf-php/pull/31!

**Comment by imbsky** [comment]
I also added CI setup to elasticsearch.

**Comment by richin13** [comment]
Sorry for the late response, `asdf-neovim` is now green

**Comment by imbsky** [comment]
@richin13 No worries! Thank you!

**Comment by vic** [comment]
`asdf-graalvm` is green again.

**Comment by imbsky** [comment]
Thanks!

**Comment by elementalvoid** [comment]
Oh dang. I missed this one -- I can help out with `asdf-brigade` (hey @blimmer!)

Since Ben was the only contributor no one @ibotta got a notification of the issue he made for us. I've set myself as a watcher and will be fixing the build very soon.

**Comment by elementalvoid** [comment]
[`asdf-brig`](https://github.com/Ibotta/asdf-brig) has been fixed!

[![GitHub Actions Status](https://github.com/ibotta/asdf-brig/workflows/Main%20workflow/badge.svg?branch=master)](https://github.com/ibotta/asdf-brig/actions)

**Comment by imbsky** [comment]
Thanks!

**Comment by joe733** [comment]
I think, this can be automated..

**Comment by jthegedus** [comment]
I don't want to remove plugins automatically. GitHub checks have intermittent failures more frequently than I enjoy.

I did update the CI so commits on the default branch tests all plugins and summarizes the information. For example:
	
```
 * Checking plugins/vendir [FAILED]
*	FAIL: Expected plugin CI badge SVG to be passing but it was not: https://github.com/vmware-tanzu/asdf-carvel/workflows/test-all/badge.svg
* Checking plugins/ytt [FAILED]
*	FAIL: Expected plugin CI badge SVG to be passing but it was not: https://github.com/vmware-tanzu/asdf-carvel/workflows/test-all/badge.svg

Plugins available: 638
Plugin build checks passed: 549
Plugin build checks failed: 89
```
>source - https://github.com/asdf-vm/asdf-plugins/actions/runs/4751694153/jobs/8441129580

I think there are ways to further automate this and report to the authors that there are issues, whether by tagging them in a thread in this repo or by opening Issues in the plugin repos. _However, I don't feel the effort is worth the reward._

Very few people seem to complain about plugins actually not working. It's only a few times a year that we replace unmaintained plugins with new forks. Although some of the repos here might not work, they give people a place to fork from and then we can update this repo with their new plugins.

This is a health check at best. It isn't, and shouldn't, be a tool for people to gauge plugin health, robustness or trust. I try to point this out as much in this repo's README, SECURITY file and core tool documentation site.

**Comment by joe733** [comment]
I agree, not removing plugins directly, instead automating the notification of failed builds, with a warning of grace time, say 3 to 6 months to fix the build. It's more of a maintenance status check, and yeah I also doubt if it's worth the effort.
